object.c

```diff
@@ -40,6 +40,10 @@ VALUE rb_cNilClass; /*!< NilClass class */
 VALUE rb_cTrueClass; /*!< TrueClass class */
 VALUE rb_cFalseClass; /*!< FalseClass class */
 
+VALUE rb_cNilClass_to_s;
+VALUE rb_cTrueClass_to_s;
+VALUE rb_cFalseClass_to_s;
```

Comment:
Should be `static` if no specific reason.

Reply:
Done.